### PR TITLE
fixing hotwire issue

### PR DESCRIPTION
**Pull request by amantu-qbit**
1. Now if anyone other than the owner takes the vehicle they'll have to hotwire the car. Or if the owner gives them keys then they'll not have to hotwire

2. Even if they lockpick they'll have to hotwire. Because they just lockpicked doors but they'll have to Hotwire regardless.

3. With config rob they'll also get the option to just take the vehicle but they'll not get the key. and will have to hotwire

4. If they rob/kill the driver they'll get the key

Note: If you don't want to keep the door open when config. rob is true then change line 368 SetVehicleDoorsLocked(entering, 1) to SetVehicleDoorsLocked(entering, 2)

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -356,7 +356,7 @@ CreateThread(function()
                 sleep = 2000
                 local plate = QBCore.Functions.GetPlate(entering)
                 QBCore.Functions.TriggerCallback('vehiclekeys:server:CheckOwnership', function(result)
-                    if not result then -- if not player owned
+                    if result ~= true then -- if not player owned
```
Comment (yamieboi) [inline client/main.lua:359]: 'result ~= true' and 'not result' is the same , would suggest changing back to changing result.

Comment (amantu-qbit) [reply client/main.lua:359]: Yeah, Ik. Changing it to not result

**Comment by BerkieBb** [comment]
This commit makes all the natives to set the vehicle's doors locked useless

**Comment by amantu-qbit** [comment]
> This commit makes all the natives to set the vehicle's doors locked useless

I know and I mentioned how to lock it

**Comment by BerkieBb** [comment]
Why remove it in the first place?

**Comment by amantu-qbit** [comment]
> Why remove it in the first place?

In customary in our region

**Pull request closed without merging**